pynq/pl.py

```diff
@@ -536,7 +536,11 @@ def client_request(cls, address=PL_SERVER_FILE,
         None
 
         """
-        cls._remote = Client(address, family='AF_UNIX', authkey=key)
+        try:
+            cls._remote = Client(address, family='AF_UNIX', authkey=key)
+        except FileNotFoundError:
+            raise ConnectionError(
+                       "Could not connect to Pynq PL server") from None
```

Comment:
@PeterOgden I think we can ignore the types of error; that is, no matter what exception occurs, we raise the ConnectionError / RuntimeError.